src/virtio/net.c

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright 2024, UNSW (ABN 57 195 873 179)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#include <libvmm/virtio/config.h>
+#include <libvmm/virtio/virtq.h>
+#include <libvmm/virtio/mmio.h>
+#include <libvmm/virtio/net.h>
+#include <libvmm/virq.h>
+#include <libvmm/util/util.h>
+#include <sddf/network/queue.h>
+
+#define REG_RANGE(r0, r1)   r0 ... (r1 - 1)
```

Comment:
Doesn't seem used.

Reply:
good spot, removed in https://github.com/au-ts/libvmm/pull/95/commits/b19f0409b85b726ac458f580b3e0d336096f1ee8